CONTRIBUTING.md

```diff
@@ -345,13 +345,32 @@ description: {
 }
 ```
 
+Changelogs may also be written in "markdown-frontmatter" format. This is useful if your
+description contains braces, which must be escaped with backslashes in `.cabal` file
+format.
+
+```markdown
+---
+synopsis: Add feature xyz
+packages: [cabal-install]
+prs: #0000
+issues: #0000 #0000
+significance: significant
+---
+
+- Detail number 1
+- Detail number 2
+
+```
+The package list must be enclosed in square brackets, but they aren't needed for `prs` or `issues`.
```

Comment:
```suggestion
The package list must be enclosed in square brackets and comma-separated, but this isn't needed for `prs` or `issues`.
```

Reply:
Sigh. I could have sworn I had added that; possibly I inadvertently removed it when I re-rewrote the markdown-frontmatter part.

Reply:
No worries!